docs/api/session.md

```diff
@@ -348,8 +348,14 @@ win.webContents.session.setCertificateVerifyProc((request, callback) => {
 
 * `handler` Function | null
   * `webContents` [WebContents](web-contents.md) - WebContents requesting the permission.  Please note that if the request comes from a subframe you should use `requestingUrl` to check the request origin.
-  * `permission` String - Enum of 'media', 'geolocation', 'notifications', 'midiSysex',
-    'pointerLock', 'fullscreen', 'openExternal'.
+  * `permission` String - Types of permission that can be requested.
+    * `media` - Allows app to access media files from a user's device with the user's consent.
+    * `geolocation` - Allows app to get the user's current location.
+    * `notifications` - Allows app to create notifications and display them in the user's system tray.
+    * `midiSysex` - Allows the usage of sysex messages in the `webmidi` API.
+    * `pointerLock` - Allows the app to directly interpret mouse movements as an input method.
+    * `fullscreen` - Allows application to run in fullscreen mode.
```

Comment:
```suggestion
    * `fullscreen` - Allows the app to run in fullscreen mode.
```

Reply:
`run in fullscreen mode` -> `enter fullscreen mode`